StantDrawing/BrushTool/BrushToolView.swift

```diff
@@ -187,27 +221,115 @@ public class BrushToolView: UIView {
     }
     
     private func configureGroupPencilSizeLayout() {
-        let origin = CGPoint(x: groupToolsView.frame.width + groupToolsView.frame.origin.x + 10, y: 0)
-        configureGroupLayout(buttons: groupPencilSizeButtons, groupView: groupPencilSizeView, groupViewOirigin: origin, addAsSubview: false)
-        groupPencilSizeView.isHidden = true
+        // Pencil Icon
+        let pencilIconView   = UIView()
         
-        setAllGroupdPencilSizeButtonsAsNotClicked()
+        let originIconView = CGPoint(x: 0, y: 0)
+        let sizeIconView   = CGSize(width: groupViewWidth, height: 60)
+        let frameIconView  = CGRect.init(origin: originIconView, size: sizeIconView)
+        
+        pencilIconView.frame              = frameIconView
+        pencilIconView.backgroundColor    = UIColor(hex: "F0F0F0")
```

Comment:
This color could be extracted to a extension, because if any changes in the future are needed, you would only change in the extension, not on where you are using it.

Reply:
Like we talked. I extracted it to a let for this View.